src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,66 @@
-export const Tabs = () => {};
+import React, { useState } from 'react';
+
+type Tab = {
+  id: string,
+  title: string,
+  content: string,
+};
+
+type Props = {
+  tabs: Tab[]
+};
+
+export const Tabs: React.FC<Props> = ({ tabs }) => {
+  const [selectedTab, setToggle] = useState(tabs[0]);
+
+  const selectedTabId: Tab = tabs.find(
+    tab => selectedTab.id === tab.id,
+  ) || tabs[0];
+
+  const chooseTab = (tab: Tab) => {
+    if (selectedTab.id !== tab.id) {
+      setToggle(tab);
+    }
+  };
+
+  return (
+
+    <>
+      <h1 className="title">
+        Selected tab is
+        {' '}
+        {selectedTab.title}
+      </h1>
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map((tab) => (
+              <li
+                key={tab.id}
+                className={selectedTab.id === tab.id ? 'is-active' : ''}
```

Comment:
use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally